files/default/sql/ddl/updates/0.9.0.sql

```diff
@@ -0,0 +1,13 @@
+ALTER TABLE jupyter_project ADD CONSTRAINT unique_hdfs_user UNIQUE (hdfs_user_id);
+
+-- Find the name of the FK
+SET @fk_name = (SELECT CONSTRAINT_NAME FROM INFORMATION_SCHEMA.KEY_COLUMN_USAGE WHERE TABLE_SCHEMA = "hopsworks" AND TABLE_NAME = "jupyter_project" AND REFERENCED_TABLE_NAME="hdfs_users");
```

Comment:
@robzor92 Just fyi, that can return multiple constraints. In the future might be safer to add referenced column name as well. We can merge it.